assets/src/edit-story/components/keyboardShortcutsMenu/index.js

```diff
@@ -35,52 +34,36 @@ import {
 /**
  * Internal dependencies
  */
-import { useHelpCenter } from '../../app';
 import { isKeyboardUser } from '../../utils/keyboardOnlyOutline';
 import DirectionAware from '../directionAware';
 import Tooltip from '../tooltip';
 import { Popup } from './popup';
 import ShortcutMenu from './shortcutMenu';
 import { TOGGLE_SHORTCUTS_MENU } from './constants';
+import { useKeyboardShortcutsMenu } from './keyboardShortcutsMenuContext';
 
 const Wrapper = styled.div``;
 
 function KeyboardShortcutsMenu() {
   const anchorRef = useRef();
   const wrapperRef = useRef();
-  const [isOpen, setIsOpen] = useState(false);
-
-  const closeMenu = useCallback(() => setIsOpen(false), [setIsOpen]);
-  const { close: closeHelpCenter } = useHelpCenter(
-    ({ actions: { close } }) => ({ close })
+  const { close, toggle, isOpen } = useKeyboardShortcutsMenu(
+    ({ actions: { close, toggle }, state: { isOpen } }) => ({
+      close,
+      toggle,
+      isOpen,
+    })
   );
 
-  const toggleMenu = useCallback(
-    (e, showMenu) => {
-      e.preventDefault();
-      setIsOpen((prevIsOpen) => {
-        const menuOpen = showMenu ?? !prevIsOpen;
-
-        if (menuOpen) {
-          closeHelpCenter();
-        }
-
-        if (isKeyboardUser() && !menuOpen) {
-          // When menu closes, return focus to toggle menu button
-          anchorRef.current.focus?.();
-        }
-
-        trackEvent('shortcuts_menu_toggled', {
-          status: menuOpen ? 'open' : 'closed',
-        });
```

Comment:
Is this being accounted for elsewhere? or is it fine to toss?

Reply:
it's getting accounted for - just in the new `toggle`